File: single-language-analysis/dataset-files/CSharp/CSharp.Tests/Tests/TestSmell055.cs
using Xunit;

public class TestSmell055
{
    [Fact(Skip = "")]
    public void TestRun_DomainInTestTld_IsConsideredNotFound()
    {
        PersistResource(Registry.Get("lol").AsBuilder().SetTldType(Registry.TldType.TEST).Build());
        Registrar registrar = PersistResource(MakeRegistrar(
            "evilregistrar", "Yes Virginia", ACTIVE));
        PersistResource(MakeDomainBase(
            "cat.lol",
            PersistResource(MakeContactResource("5372808-ERL", "Goblin Market", "lol@cat.lol")),
            PersistResource(MakeContactResource("5372808-IRL", "Santa Claus", "BOFH@cat.lol")),
            PersistResource(MakeContactResource("5372808-TRL", "The Raven", "bog@cat.lol")),
            PersistResource(MakeHostResource("ns1.cat.lol", "1.2.3.4")),
            PersistResource(MakeHostResource("ns2.cat.lol", "bad:f00d:cafe::15:beef")),
            registrar));
        PersistSimpleResources(MakeRegistrarContacts(registrar));
        NewWhoisAction("domain cat.lol\r\n").Run();
        Assert.Equal(200, Response.GetStatus());
        Assert.Equal(LoadFile("whois_action_domain_not_found.txt"), Response.GetPayload());
    }
}

File: single-language-analysis/dataset-files/python/test_smell_055.py
import pytest

class TestWhoisAction:
    @pytest.mark.skip
    def test_smell_055(self):
        persist_resource(registry.get("lol").as_builder().set_tld_type(registry.TldType.TEST).build())
        registrar = persist_resource(make_registrar(
            "evilregistrar", "Yes Virginia", ACTIVE))
        persist_resource(make_domain_base(
            "cat.lol",
            persist_resource(make_contact_resource("5372808-ERL", "Goblin Market", "lol@cat.lol")),
            persist_resource(make_contact_resource("5372808-IRL", "Santa Claus", "BOFH@cat.lol")),
            persist_resource(make_contact_resource("5372808-TRL", "The Raven", "bog@cat.lol")),
            persist_resource(make_host_resource("ns1.cat.lol", "1.2.3.4")),
            persist_resource(make_host_resource("ns2.cat.lol", "bad:f00d:cafe::15:beef")),
            registrar))
        persist_simple_resources(make_registrar_contacts(registrar))
        new_whois_action("domain cat.lol\r\n").run()
        assert response.get_status() == 200
        assert response.get_payload() == load_file("whois_action_domain_not_found.txt")